src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -324,6 +326,22 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode)
                                         ->defaultValue([])
                                         ->prototype('scalar')->end()
                                     ->end()
+                                    ->arrayNode('dispatched_events')
```

Comment:
Could definitely use input on configuration here, this is my first time working on the Symfony code base and this was somewhat confusing.

I ran into issues with a number of things such as expressing an empty array in the XML configuration format, as well as having a default null value for arrayNodes.

I have worked around this by specifying `all` and `none` as allowed values for this configuration option however these are mostly hidden from the end user. The only exception is with the XML configuration where you need to specify `none` to remove all events from being fired.